apis/marathon_api.py

```diff
@@ -122,6 +123,27 @@ def get_all_marathon_prompts():
         return jsonify(results)
 
 
+@marathon_api.get('/get_prompts')
+def get_marathon_prompts():
+
+    limit = int(request.args.get('limit', 5))
+
+    args = {'p_limit': limit}
+
+    query = """
+    SELECT prompt_id, start, initcheckpoints, checkpoints, cmty_added_by, cmty_anonymous, cmty_submitted_time, username FROM marathonprompts
+    LEFT JOIN users
+    ON users.user_id = marathonprompts.cmty_added_by
+    ORDER BY prompt_id DESC
+    LIMIT %(p_limit)s
+    """
+
+    db = get_db()
+    with db.cursor(cursor=DictCursor) as cursor:
+        cursor.execute(query, args)
+        results = cursor.fetchall()
+
+        return jsonify(results)
```

Comment:
Is this just a special case of the archive endpoint? If so you might just want to use  the `prompts.get_archive_prompts` to reduce code duplication? Or are there fields that aren't there?

Reply:
Fixed. This was duplicate code